src/Compilers/VisualBasic/Portable/Binding/Binder_Statements.vb

```diff
@@ -4144,27 +4144,27 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         ''' Checks if a given symbol is a function that takes no parameters.
         ''' </summary>
         Private Shared s_isFunctionWithoutArguments As Func(Of Symbol, Boolean) = Function(sym)
-                                                                                    If sym.Kind = SymbolKind.Method Then
-                                                                                        Dim method = DirectCast(sym, MethodSymbol)
-                                                                                        Return Not method.IsSub() AndAlso
-                                                                                               Not method.IsGenericMethod AndAlso
-                                                                                               method.CanBeCalledWithNoParameters
-                                                                                    End If
-                                                                                    Return False
-                                                                                End Function
+                                                                                      If sym.Kind = SymbolKind.Method Then
```

Comment:
@AdamSpeight2008 we've been over this.  The spacing requirement for dotnet Foundation projects is 4 spaces.